client-tools/Shell/V8ClientConnection.cpp

```diff
@@ -3592,12 +3592,7 @@ void V8ClientConnection::initServer(v8::Isolate* isolate,
                                WrapV8ClientConnection(isolate, this));
   TRI_AddGlobalVariableVocbase(isolate,
                                TRI_V8_ASCII_STRING(isolate, "SYS_IS_V8_BUILD"),
-#ifndef USE_V8
-                               v8::False(isolate)
-#else
-                               v8::True(isolate)
-#endif
-  );
+                               v8::False(isolate));
```

Comment:
I think we should completely remove this variable and fix up the tests, but this should be done in a separate PR!